src/main/java/jenkins/scm/impl/subversion/SubversionSCMSource.java

```diff
@@ -136,15 +139,18 @@ public SubversionSCMSource(String id, String remoteBase, String credentialsId, S
         setCredentialsId(credentialsId);
         setIncludes(StringUtils.defaultIfEmpty(includes, DescriptorImpl.DEFAULT_INCLUDES));
         setExcludes(StringUtils.defaultIfEmpty(excludes, DescriptorImpl.DEFAULT_EXCLUDES));
+        this.additionalCredentials =  new ArrayList<AdditionalCredentials>();
```

Comment:
Usage of this collection is ineffective if you do not plan to modify it on the flight. Use `Collections.emptyList()` instead in the code